systemtest/gencorpora/catbulk.go

```diff
@@ -113,18 +171,31 @@ func handleReq(writer io.Writer) http.HandlerFunc {
 
 			mockResp := esutil.BulkIndexerResponse{}
 			scanner := bufio.NewScanner(reader)
+			var stat docsStat
 			for count := 0; scanner.Scan(); count++ {
-				fmt.Fprintln(writer, scanner.Text())
+				n, err := fmt.Fprintln(writer, scanner.Text())
```

Comment:
Maybe panic if there's an error? We wouldn't expect an error except in case of something exceptional like disk full, right?

Reply:
I have reworked the PR and now I am logging the errors in this case and discarding the whole request. The error log should be visible in APM-Server logs also along with the `gencorpora` logs, let me know if it makes sense.

Reply:
Looks good :)